audiomentations/augmentations/transforms.py

```diff
@@ -1151,7 +1154,236 @@ def apply(self, samples, sample_rate):
+        
+        audio_segment = pydub.AudioSegment(
+            int_samples.tobytes(),
+            frame_rate=sample_rate,
+            sample_width=int_samples.dtype.itemsize,
+            channels=1,
+        )
+
+        audio_segment = pydub.effects.low_pass_filter(
+            audio_segment, self.parameters["cutoff_freq"]
+        )
+        samples = convert_int16_samples_to_float(np.array(audio_segment.get_array_of_samples()))
+        return samples
+
+
+class HighPassFilter(BaseWaveformTransform):
+    """
+    Apply high-pass filtering to the input audio.
+    """
+
+    supports_multichannel = False
+    requires_sample_rate = True
+
+    def __init__(
+        self,
+        min_cutoff_freq=20,
+        max_cutoff_freq=2400,
+        p: float = 0.5
+    ):
+        """
+        :param min_cutoff_freq: Minimum cutoff frequency in hertz
+        :param max_cutoff_freq: Maximum cutoff frequency in hertz
+        :param p: The probability of applying this transform
+        """
+        super().__init__(p)
+
+        self.min_cutoff_freq = min_cutoff_freq
+        self.max_cutoff_freq = max_cutoff_freq
+        if self.min_cutoff_freq > self.max_cutoff_freq:
+            raise ValueError("min_cutoff_freq must not be greater than max_cutoff_freq")
+
+    def randomize_parameters(
+        self, samples: np.array, sample_rate: int = None
+    ):
+        """
+        :params samples: (num_channels, num_samples)
+        """
+        super().randomize_parameters(samples, sample_rate)
+
+        self.parameters["cutoff_freq"] = np.random.uniform(
+            low=self.min_cutoff_freq,
+            high=self.max_cutoff_freq
+        )
+
+    def apply(self, samples: np.array, sample_rate: int = None):
+        try:
+            import pydub
+        except ImportError:
+            print(
+                "Failed to import pydub. Maybe it is not installed? "
+                "To install the optional pydub dependency of audiomentations,"
+                " do `pip install audiomentations[extras]` instead of"
+                " `pip install audiomentations`",
+                file=sys.stderr,
+            )
+            raise
+            
+        assert len(samples.shape) == 1
+        assert samples.dtype == np.float32
+        
+        int_samples = convert_float_samples_to_int16(samples)
+        
+        audio_segment = pydub.AudioSegment(
+            int_samples.tobytes(),
+            frame_rate=sample_rate,
+            sample_width=int_samples.dtype.itemsize,
+            channels=1,
+        )
 
+        audio_segment = pydub.effects.high_pass_filter(
+            audio_segment, self.parameters["cutoff_freq"]
+        )
+        samples = convert_int16_samples_to_float(np.array(audio_segment.get_array_of_samples()))
+        return samples
+
+
+class BandPassFilter(BaseWaveformTransform):
+    """
+    Apply band-pass filtering to the input audio.
+    """
+
+    supports_multichannel = True
```

Comment:
This must be set to False to be aligned with the implementation

Reply:
Fixed.